docs/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,129 @@
+# Contributor Covenant Code of Conduct
```

Comment:
As this is already in the root of the repo I'm not sure we need to include it again in the `docs/` directory?

Reply:
Yeah sounds good!